gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/matching/common/query/predicates/CNFElement.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of Gradoop.
+ *
+ * Gradoop is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Gradoop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Gradoop. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.gradoop.flink.model.impl.operators.matching.common.query.predicates;
+
+import org.gradoop.flink.model.impl.operators.matching.common.query.predicates.wrappers
+  .expressions.ComparisonWrapper;
```

Comment:
just scroll through the changed files and check where this is the case and change it. I'll skip comments from now on :)